_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -0,0 +1,81 @@
+
+**Mission**. We've defined an initial mission: The US-RSE Association is centered around three main goals. We aim to target our activities and actions to serving these three aspects:
+
+**Community**: We seek to provide a coherent association of those who identify with the role (not necessarily title) of Research Software Engineer based on our inclusive definition. This group aims to provide the members of the community the ability to share knowledge, professional connections, and resources.
+
+**Advocacy**: We aim to promote RSEs’ impact on research, highlighting the increasingly critical and valuable role RSEs serve.
+
+**Resources**: We strive to provide useful resources to multiple demographics. For current and future RSEs we strive to provide technical and career development resources to support their professional development. We aim to provide access to information and material to support the establishment and expansion of RSE positions and groups within the research ecosystem including the material for making business justifications for RSEs.
+
+**Logo, website, Slack**. We've created a [US-RSE logo](https://github.com/USRSE/logo) and a [US-RSE website](https://us-rse.org/) that's hosted on [GitHub](https://github.com/USRSE/usrse.github.io) and has an [open method for people to contribute](https://github.com/USRSE/usrse.github.io/blob/master/README.md) jobs, events, and other content. As part of the website, we build a map of RSE member locations. We've also created an RSE Slack team with channels for different topics of interest that enable US-RSE members to chat with each other. As of 22 April 2020, the Slack team, which includes both members and others who are interested in US-RSE issues, has 522 users. 
+
+![US-RSE member locations]({{ site.baseurl }}/assets/posts/2020-04-28-member-map.png "US-RSE member locations")
+
+**Code of Conduct**. In September 2019, we created a [Code of Conduct](https://us-rse.org/code-of-conduct/) for the US-RSE Association to encourage participation in our community, and to make clear what isn't acceptable.
+
+**Job board**. One element of the website is a [job board](https://us-rse.org/jobs/) that lists open jobs of interest to RSEs. New jobs can be contributed by making a pull request to the GitHub source for the website like other types of contributions, or by asking the website team to do this via the RSE slack team. Since this board was created in May 2019, 35 jobs have been posted and as of 22 April 2020, 6 are active. 
+
+**Twitter and newsletters**. We created a [US-RSE twitter account](https://twitter.com/us_rse) in June 2019, which as of 22 April 2020 has 500 followers and has made 119 tweets, and a much larger number of retweets. Additionally, we've been writing a [newsletter](https://us-rse.org/newsletters/) about every 2-3 months, which is mailed out to all RSE members and posted on our website.
```

Comment:
mailed out to all RSE members -> mailed out to all US-RSE members.

Reply:
will fix

Reply:
fixed